code-challenges.test.js

```diff
@@ -15,45 +15,114 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
-var num2 = 0
-// Expected output: "0 is divisible by three"
-var num3 = -7
-// Expected output: "-7 is not divisible by three"
+describe("divisibleBy3", () => {
+          var num1 = 15
+           var num2 = 0
+          var num3 = -7
+// I am describing the goal of the function which is to be able to take the numbers in the variable and check if the numbers are divisble by three
+
+    it("logs decides if the number is evenly divisble by three or not", () => {
+
+      expect(divisibleBy3(num1)).toEqual(`${number1} is divisible by three.`)
+      expect(divisibleBy3(num2)).toEqual(`${number2} is divisible by three.`)
+      expect(divisibleBy3(num3)).toEqual(`${number3} is not divisible by three.`)
+    })
+  })
+// I am expicting to get the result in each numbers of the variable.
```

Comment:
Good job moving the variables inside the test. I am a little confused about the difference between num1 and number1.